websites/P/dist/metadata.json

```
@@ -0,0 +1,26 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "Patreon",
+    "author": {
+      "name": "rexulec",
+      "id": "709376748206293002"
+  },
+  "category": "socials",
+  "logo": "https://i.imgur.com/d0YtwcF.png",
+  "thumbnail": "https://i.imgur.com/hT1CAqd.png",
+  "version": "1.0.0",
+  "color": "#f86754",
+  "tags": [
+    "subs",
+    "paid",
+    "donate",
+    "subscription",
+    "support"
+  ],
+    "url": ["www.patreon.com", "patreon.com"],
+    "description": {
+      "en": "Patreon, is an Internet-based platform that allows creators to set up their own subscription content services.",
+      "tr": "Patreon, içerik oluşturucularına kendi abonelik içeriği hizmetlerini kurmalarına izin veren, İnternet tabanlı bir platformdur.",
+      "de": "Patreon,ist eine internetbasierte Plattform, mit der Entwickler ihre eigenen Abonnementinhaltsdienste einrichten können."  
+    }
```

```suggestion
    "description": {
      "en": "Patreon is a membership platform that makes it easy for artists and creators to get paid. Join over 200,000+ creators earning salaries from over 4 million monthly patrons.",
      "nl": "Patreon is een lidmaatschapsplatform waarmee artiesten en makers gemakkelijk betaald kunnen worden. Sluit je aan bij meer dan 200.000+ videomakers die salarissen verdienen van meer dan 4 miljoen maandelijkse patrons."
    }
```
Use the official description, added it for Dutch, don't speak German or Turkish so can't translate it to those...